appveyor.yml

```diff
@@ -49,6 +49,7 @@ skip_commits:
 branches:
   only:
     - develop
+    - 12.Next

```

Comment:
Does this mean that appveyor will build and publish nuget packages from this branch?
We don't want that, but it would be nice to have a PR build.